src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -0,0 +1,348 @@
+    const user = this.personDetails;
+    // tslint:disable-next-line: one-variable-per-declaration
+    let chat, email, phone;
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._chatUser}>
+          ${svgHelper.getSVG('chat', '#666666')}
+        </div>
+      `;
+    }
+    if (getEmailFromGraphEntity(user)) {
+      email = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._emailUser}>
+          ${svgHelper.getSVG('email', '#666666')}
+        </div>
+      `;
+    }
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = html`
+        <div @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState} @click=${this._callUser}>
+          ${svgHelper.getSVG('phone', '#666666')}
+        </div>
+      `;
+    }
+
+    if (this.isExtended === true) {
+      return html``;
+    } else {
+      return html`
+        ${chat} ${email} ${phone}
+      `;
+    }
+  }
+
+  private renderAdditionalDetails() {
+    const user = this.personDetails;
+
+    // tslint:disable-next-line: one-variable-per-declaration
+    let phone, email, location, chat;
+
+    if ((user as MicrosoftGraph.User).businessPhones && (user as MicrosoftGraph.User).businessPhones.length > 0) {
+      phone = html`
+        <div class="details-icon" @click=${this._callUser}>
+          ${svgHelper.getSVG('phone-small', '#666666')}
+          <span class="link-subtitle data" @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState}
+            >${(user as MicrosoftGraph.User).businessPhones[0]}</span
+          >
+        </div>
+      `;
+    }
+
+    if (getEmailFromGraphEntity(user)) {
+      email = html`
+        <div class="details-icon" @click=${this._emailUser}>
+          ${svgHelper.getSVG('email-small', '#666666')}
+          <span class="link-subtitle data" @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState}
+            >${getEmailFromGraphEntity(user)}</span
+          >
+        </div>
+      `;
+    }
+
+    if ((user as MicrosoftGraph.User).mailNickname) {
+      chat = html`
+        <div class="details-icon" @click=${this._chatUser}>
+          ${svgHelper.getSVG('chat-small', '#666666')}
+          <span class="link-subtitle data" @mouseout=${this._unsetMouseOverState} @mouseover=${this._setMouseOverState}
+            >${(user as MicrosoftGraph.User).mailNickname}</span
+          >
+        </div>
+      `;
+    }
+
+    if (user.officeLocation) {
+      location = html`
+        <div class="details-icon">
+          ${svgHelper.getSVG('location-small', '#666666')}<span class="normal-subtitle data"
+            >${user.officeLocation}</span
+          >
+        </div>
+      `;
+    }
+
+    if (this.isExtended === true) {
```

Comment:
same as previous method - no need to do above if not expanded

Reply:
Refactor: e6978a7